pom.xml

```diff
@@ -738,7 +737,7 @@
       <dependency>
         <groupId>com.fasterxml.jackson.core</groupId>
         <artifactId>jackson-databind</artifactId>
-        <version>${jackson-databind.version}</version>
```

Comment:
If jackson-databind is part of the jackson-bom, then additional dependency is not needed, can you check?

Reply:
Removed